js/subassembly/dialog.js

```diff
@@ -0,0 +1,148 @@
+                params.content
+            ])
+        }
+        return ''
+    }
+    const el = $el("div.bizyair-new-dialog", {
+        parent: document.body,
+        id,
+        style: { zIndex: 10000 + document.querySelectorAll('.bizyair-new-dialog').length },
+        onclick: function () {
+            if (params.closeOnClickModal) {
+                removeDialog(this)
+            }
+        },
+    }, [
+        $el("div.bizyair-dialog-content", {
+            style,
+            onclick: function (e) {
+                e.stopPropagation();
+            }
+        }, [
+            (params.title ? $el("p.bizyair-new-dialog-title", { }, [params.title]) : ''),
+            setContent(),
+            $el('div.bizyair-new-dialog-footer', {}, [
+                (params.yesText ? $el("button.bizyair-new-dialog-btn", {
+                    type: "button",
+                    textContent: params.yesText,
+                    id: params.yesId ? params.yesId : '',
+                    onclick: async () => {
+                        if (params.onYes) {
+                            const res = await params.onYes();
+                            if (!res) {
+                                return false
+                            }
+                        }
+                        removeDialog(document.getElementById(id))
+                    }
+                }): ''),
+                (params.neutralText ? $el("button.bizyair-new-dialog-btn", {
+                    type: "button",
+                    textContent: params.neutralText,
+                    id: params.neutralId ? params.neutralId : '',
+                    onclick: async () => {
+                        if (params.onNeutral) {
+                            const res = await params.onNeutral();
+                            if (!res) {
+                                return false
+                            }
+                        }
+                        removeDialog(document.getElementById(id))
+                    }
+                }): ''),
+                (params.noText ? $el("button.bizyair-new-dialog-btn", {
+                    type: "button",
+                    textContent: params.noText,
+                    onclick: async () => {
+                        if (params.onNo) {
+                            await params.onNo();
+                        }
+                        removeDialog(document.getElementById(id))
+                    }
+                }): '')
+            ]),
+        ])
+    ]);
+    const fnEscapeClose = async function (e) {
+        if (e.key === "Escape") {
+            const topDialog = dialogStack[dialogStack.length - 1];
+
+            if (topDialog === el) {
+                if (params.onNo) {
+                    await params.onNo();
+                }
+                removeDialog(el);
+            }
+        }
+    }
+    dialogStack.push(el);
+
+    if (!params.onEsape) {
```

Comment:
```suggestion
    if (!params.onEscape) {
```

应该是拼写笔误吧，如果有相关地方也需要修改下。